app/assets/stylesheets/provider/_vertical_nav.scss

```diff
@@ -1,6 +1,7 @@
 .pf-c-page__sidebar {
   text-align: left;
   text-decoration: none;
+  color: #fff;
```

Comment:
In assets managed by Rails under `app/assets/stylesheets` you can import (or add if missing) the colors as vars from `app/assets/stylesheets/provider/_colors.scss` using: 

`@import 'provider/colors';`